src/main/java/TaskManager.java

```diff
@@ -0,0 +1,91 @@
+public class TaskManager extends Duke {
+    protected String command;
+    protected String description;
+    protected String date;
+
+    public TaskManager(String command, String description, String date) {
+        this.command = command;
+        this.description = description;
+        this.date = date;
+    }
+
+    /**
+     * Handles commands from the input based on the command word, which is the first word from input.
+     */
+    public void handleCommand() {
+        switch (command) {
+        case "todo":
+            // Labels task as T
+            if (listIndex < listSize) {
+                taskList[listIndex] = new ToDo(description);
+                listIndex++;
+                printAddedMessage();
+            } else {
+                printFailedToAddMessage();
+            }
+            break;
+        case "event":
+            // Labels task as E and also takes in a date
+            if (listIndex < listSize) {
+                taskList[listIndex] = new Event(description, date);
+                listIndex++;
+                printAddedMessage();
+            } else {
+                printFailedToAddMessage();
+            }
+            break;
+        case "deadline":
+            // Labels task as D and also takes in a date
+            if (listIndex < listSize) {
+                taskList[listIndex] = new Deadline(description, date);
+                listIndex++;
+                printAddedMessage();
+            } else {
+                printFailedToAddMessage();
+            }
+            break;
+        case "done":
+            // Marks task x as done where x is the index.
+            int index = Integer.parseInt(description);
+            if (index <= listIndex) {
```

Comment:
Create a boolean to explain what is this. Hard to understand this condition statement and it's meaning